test/test_tls.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'helper'
+require 'fluent/tls'
+
+class UniqueIdTest < Test::Unit::TestCase
+  TEST_TLS1_1_CASES = {
+    'New TLS v1.1' => :'TLS1_1',
+    'Old TLS v1.1' => :'TLSv1_1',
+  }
+  TEST_TLS1_2_CASES = {
+    'New TLS v1.2' => :'TLS1_2',
+    'Old TLS v1.2' => :'TLSv1_2'
+  } 
+  TEST_TLS_CASES = TEST_TLS1_1_CASES.merge(TEST_TLS1_2_CASES)
+
+  sub_test_case 'constants' do
```

Comment:
Do these tests need? It looks that they do nothing...

Reply:
These constants are used by plugins.
These tests are for preventing unexpected value changes.